gradlew

```diff
@@ -42,11 +42,6 @@ case "`uname`" in
     ;;
 esac
 
-# For Cygwin, ensure paths are in UNIX format before anything is touched.
-if $cygwin ; then
-    [ -n "$JAVA_HOME" ] && JAVA_HOME=`cygpath --unix "$JAVA_HOME"`
-fi
-
```

Comment:
[Generating the gradle wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html) automatically generates `./gradlew` and `gradle.bat`, I'm not sure what the changes do, but I know it's more up-to-date and Gradle 2.11